docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -328,6 +328,226 @@ And create it:
 kubectl create -f my-crontab.yaml
 crontab "my-new-cron-object" created
 ```
+
+### Subresources
+
+Custom resources support `/status` and `/scale` subresources.
+This feature is __alpha__ in v1.10 and may change in backward incompatible ways.
+
+Enable this feature using the `CustomResourceSubresources` feature gate on
+the [kube-apiserver](/docs/admin/kube-apiserver):
+
+```
+--feature-gates=CustomResourceSubresources=true
+```
+
+When the `CustomResourceSubresources` feature gate is enabled, only the `properties` construct
+is allowed in the root schema for custom resource validation.
+
+The status and scale subresources can be optionally enabled by
+defining them in the CustomResourceDefinition.
+
+#### Status subresource
+
+When the status subresource is enabled, the `/status` subresource for the custom resource is exposed.
+
+- The status and the spec stanzas are represented by the `.status` and `.spec` JSONPaths respectively inside of a custom resource.
+- `PUT` requests to the `/status` subresource take a custom resource object and ignore changes to anything except the status stanza.
+- `PUT` requests to the `/status` subresource only validate the status stanza of the custom resource.
+- `PUT`/`POST`/`PATCH` requests to the custom resource ignore changes to the status stanza.
+- Any changes to the spec stanza increments the value at `.metadata.generation`.
+
+#### Scale subresource
+
+When the scale subresource is enabled, the `/scale` subresource for the custom resource is exposed.
+The `autoscaling/v1.Scale` object is sent as the payload for `/scale`.
+
+To enable the scale subresource, the following values are defined in the CustomResourceDefinition.
+
+1. `SpecReplicasPath` defines the JSONPath inside of a custom resource that corresponds to `Scale.Spec.Replicas`.
+
+- It is a required value.
+- Only JSONPaths under `.spec` and with the dot notation are allowed.
+- If there is no value under the `SpecReplicasPath` in the custom resource,
+the `/scale` subresource will return an error on GET.
+
+2. `StatusReplicasPath` defines the JSONPath inside of a custom resource that corresponds to `Scale.Status.Replicas`.
```

Comment:
@heckj Any idea why the numbers 2, 3,.. are also showing up as 1 in the webpage?

https://deploy-preview-7439--kubernetes-io-vnext-staging.netlify.com/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#scale-subresource

Reply:
@nikhita The jekyll markdown support for ordered lists treats the `1.` as a reference for an ordered list - and works fine when you don’t have sub-bullets, but it appears to be a limitation that you can’t mix ordered and unordered list types - for example, the bullets - or it resets the counting for the ordered lists, which is what you’re seeing

You could make them all unordered lists (getting rid of the numbers and using `-`) or all ordered, and you’d likely be fine either way

Reply:
@heckj Thanks! I'll make them unordered lists. :+1: